### DISCUSSION TITLE
Support for Spotify login via 3rd party accounts.

### DISCUSSION THREAD

**Discussion by Trigun** [music-assistant/support#448] [Feature requests and ideas]
Possibility to login to Spotify via eg. Your Facebook account.

**Comment by Trigun** [comment]
@marcelveldt, any idea when this might be on the agenda? Moving a current Spotify account to a new Spotify account with only a login via Spotify is a very big struggle. Many thanks!

**Comment by galahad22** [comment]
Had the same issue. There is a workaround though: You can create a Device password in Spotify account settings that behaves like an original spotify account: https://www.spotify.com/de/account/set-device-password/

**Reply by marcelveldt** [reply]
Thanks! This is a great to have on the [FAQ section](https://github.com/music-assistant/hass-music-assistant/discussions/categories/q-a-faq) for others.

**Comment by Trigun** [comment]
This is working indeed, thanks a lot!
Not sure if this is the solution but it might take it down on the backlog for items to solve? But that's up to @marcelveldt

**Reply by marcelveldt** [reply]
It depends on the number of votes on this request. Now that there is a valid workaround with the device password I do not see a real need for this though but maybe I'm overseeing something.

**Comment by TomRomeo** [comment]
The proposed answer to set a device password doesn't seem to work anymore. I created my account with google and when I want to add a device password I get the message: 'The password could not be set'

**Comment by OzGav** [comment]
This has been overtaken but Spotify’s changed login requirements
